src/app/api/oauth/next-auth.ts

```diff
@@ -0,0 +1,17 @@
+import NextAuth from 'next-auth';
+import Auth0 from 'next-auth/providers/auth0';
+
+const nextAuth = NextAuth({
+  providers: [
+    Auth0({
+      clientId: process.env.AUTH0_CLIENT_ID || '',
+      clientSecret: process.env.AUTH0_CLIENT_SECRET || '',
+      issuer: process.env.AUTH0_ISSUER,
```

Comment:
please replace `process.env.AUTH0_CLIENT_ID` by `getServerConfig`, we want to make all ENVs are imported from `src/config/server.ts`  or `src/config/client.ts`.

and then we can see the all env type defined in here: https://github.com/lobehub/lobe-chat/blob/966992aeb1c4b2f7dfcdbe900c4c91fcf0e1962a/src/config/server.ts#L6-L25

Reply:
Thanks for the guidance, I've changed the code and made sure to use `getServerConfig` to standardise the code style!